__tests__/series.test.ts

```diff
@@ -424,120 +424,142 @@ describe("series", () => {
-    ${"len"}             | ${pl.Series([1, 2, 3, 4, 5]).len()}                                                        | ${5}
-    ${"limit"}           | ${pl.Series([1, 2, 3, 4, 5, 5, 5]).limit(2)}                                               | ${pl.Series([1, 2])}
-    ${"max"}             | ${pl.Series([-1, 10, 3]).max()}                                                            | ${10}
-    ${"mean"}            | ${pl.Series([1, 1, 10]).mean()}                                                            | ${4}
-    ${"median"}          | ${pl.Series([1, 1, 10]).median()}                                                          | ${1}
-    ${"min"}             | ${pl.Series([-1, 10, 3]).min()}                                                            | ${-1}
-    ${"nChunks"}         | ${pl.Series([1, 2, 3, 4, 4]).nChunks()}                                                    | ${1}
-    ${"nullCount"}       | ${pl.Series([1, null, null, 4, 4]).nullCount()}                                            | ${2}
-    ${"peakMax"}         | ${pl.Series([9, 4, 5]).peakMax()}                                                          | ${pl.Series([true, false, true])}
-    ${"peakMin"}         | ${pl.Series([4, 1, 3, 2, 5]).peakMin()}                                                    | ${pl.Series([false, true, false, true, false])}
-    ${"quantile"}        | ${pl.Series([1, 2, 3]).quantile(0.5)}                                                      | ${2}
-    ${"rank"}            | ${pl.Series([1, 2, 3, 2, 2, 3, 0]).rank("dense")}                                          | ${pl.Series("", [2, 3, 4, 3, 3, 4, 1], pl.UInt32)}
-    ${"rename"}          | ${pl.Series([1, 3, 0]).rename("b")}                                                        | ${pl.Series("b", [1, 3, 0])}
-    ${"rollingMax"}      | ${pl.Series([1, 2, 3, 2, 1]).rollingMax(2)}                                                | ${pl.Series("", [null, 2, 3, 3, 2], pl.Float64)}
-    ${"rollingMin"}      | ${pl.Series([1, 2, 3, 2, 1]).rollingMin(2)}                                                | ${pl.Series("", [null, 1, 2, 2, 1], pl.Float64)}
-    ${"rollingSum"}      | ${pl.Series([1, 2, 3, 2, 1]).rollingSum(2)}                                                | ${pl.Series("", [null, 3, 5, 5, 3], pl.Float64)}
-    ${"rollingMean"}     | ${pl.Series([1, 2, 3, 2, 1]).rollingMean(2)}                                               | ${pl.Series("", [null, 1.5, 2.5, 2.5, 1.5], pl.Float64)}
-    ${"rollingVar"}      | ${pl.Series([1, 2, 3, 2, 1]).rollingVar(2)[1]}                                             | ${0.5}
-    ${"rollingMedian"}   | ${pl.Series([1, 2, 3, 3, 2, 10, 8]).rollingMedian({ windowSize: 2 })}                      | ${pl.Series([null, 1.5, 2.5, 3, 2.5, 6, 9])}
+    name                 | actual                                                                                         | expected
```

Comment:
I think we can just throw all of the tests in `expr.test`

This is what we are doing with other ones. (testing series + expr) at the same time

example: 
```ts
  test("contains:regex", () => {
    const df = pl.DataFrame({
      a: ["Foo", "foo", "FoO"],
    });

    const re = new RegExp("foo", "i");
    const expected = pl.DataFrame({
      a: ["Foo", "foo", "FoO"],
      contains: [true, true, true],
    });
    const seriesActual = df.getColumn("a").str.contains(re).rename("contains");
    const actual = df.withColumn(col("a").str.contains(re).as("contains"));
    expect(actual).toFrameEqual(expected);
    expect(seriesActual).toSeriesEqual(expected.getColumn("contains"));
  });

Reply:
So you want to move 140 tests to `expt.test`?

Reply:
no, just the new ones for ewm